### PR TITLE
SOF-6720:  reformat `qe_scf_calculation.ipynb` notebook

### PR DESCRIPTION
**Pull request by mrakitin**
Small fixes:
- correct the name of the package which provides `pip-compile`
- reformat the `qe_scf_calculation.ipynb` notebook with `black-jupyter`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ localhost = [
 ]
 dev = [
```
Comment (pranabdas) [inline pyproject.toml:29]: @mrakitin shall we describe the developer setup in the README under contribute? Such as we need to install dev dependencies: `pip install -e ."[dev]"`, and how to setup black-jupyter formatter.

Comment (mrakitin) [reply pyproject.toml:29]: Good idea, @pranabdas!

Comment (mrakitin) [reply pyproject.toml:29]: @pranabdas, I've added a step in the "Contribute" section explaining this: [753b669](https://github.com/Exabyte-io/api-examples/pull/80/commits/753b669eed1dff02ac399ad5e707d7961dd40a60). Please have a look.

File: README.md [hunk 2]
```diff
@@ -79,15 +79,25 @@ NOTE: The Materials Project API key should be obtained from [https://legacy.mate
...
 
 If you would like to add new examples or adjust existing ones, please consider the following:
 
 1. Put examples into the corresponding directories by domain.
 
-2. Walk the readers through the examples by providing step-by-step explanation similar to [this](examples/material/get_materials_by_formula.ipynb).
+2. Walk the readers through the examples by providing step-by-step explanation similar to [this](examples/material/get_materials_by_formula.ipynb) example.
 
-3. We use post-save hooks to automatically convert notebooks to python scripts. See [config](examples/config.py) file for more information. In order to facilitate code review, we exclude notebook sources from version control and store them in Git LFS [[3](#links)]. Please follow this convention.
+3. We use post-save hooks to automatically convert notebooks to python scripts. See [config](examples/config.py) file for more information. In order to facilitate code review, we exclude notebook sources in the `other/` directory from version control and store them in Git LFS [[3](#links)]. Please follow this convention.
```
Comment (mrakitin) [inline README.md:90]: I think we discussed with @timurbazhirov that the Git LFS approach is not used for the notebook .ipynb files in the `examples/` directory to facilitate the loading of the notebooks in Google Colab. I added a note about it here that it's applicable to the notebooks in the `other/` directory.

**Comment by mrakitin** [comment]
@pranabdas, @timurbazhirov, this is ready for review.

I am still observing Error 500, but it can/should be resolved separately:

```py
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
[<ipython-input-5-10fdecc9a4a6>](https://localhost:8080/#) in <cell line: 1>()
----> 1 response = endpoint.create(BODY)

4 frames
[/usr/local/lib/python3.10/dist-packages/requests/models.py](https://localhost:8080/#) in raise_for_status(self)
    958 
    959         if http_error_msg:
--> 960             raise HTTPError(http_error_msg, response=self)
    961 
    962     def close(self):

HTTPError: 500 Server Error: Internal Server Error for url: [https://platform.mat3ra.com:443/api/2018-10-01/workflows/create](https://platform.mat3ra.com/api/2018-10-01/workflows/create)
```

**Comment by pranabdas** [comment]
OK, let's bring over these updates to the other branch and see here https://github.com/Exabyte-io/api-examples/pull/79

**Pull request merged**